cnv/cnv_users_guide/cnv-dashboard.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-dashboard"]
+= Using the CNV Dashboard to get cluster information
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-dashboard
+toc::[]
+
+The Web Console is part of the {CNVProductName} interface.
```

Comment:
1. "Web Console" shouldn't be capitalized
2. Did you mean to refer to the dashboard here (as opposed to the web console)?

Reply:
Simplified/clarified topic to: 

Access the overview dashboard, which captures high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-title} web console.

The overview dashboard provides various cluster information, captured in individual dashboard cards.